### PR TITLE
Add hyperfine

### PR DESCRIPTION
**Pull request by The-King-of-Toasters**
Same developer as fd and bat, so I just used build.sh from fd.

### CODE REVIEW

File: build/hyperfine/build.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    logcmd cp target/release/$PROG $DESTDIR/$PREFIX/bin/fd || logerr "cp failed"
+
+    logcmd mkdir -p $DESTDIR/$PREFIX/share/man/man1
+    logcmd cp doc/$PROG.1 $DESTDIR/$PREFIX/share/man/man1/ || logerr "cp failed"
+
+    popd >/dev/null
+}
+
+init
+download_source $PROG $PROG $VER
```
Comment (citrus-it) [inline build/hyperfine/build.sh:55]: We'll put the source on our mirror with the name it comes off the github releases page, so can you adjust this to the following?
```suggestion
download_source $PROG v$VER
```

File: build/hyperfine/build.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    logcmd cargo build $args || logerr "build failed"
+    popd >/dev/null
+}
+
+install() {
+    logmsg "Installing"
+    pushd $TMPDIR/$BUILDDIR >/dev/null
+
+    logcmd mkdir -p $DESTDIR/$PREFIX/bin
+    logcmd cp target/release/$PROG $DESTDIR/$PREFIX/bin/fd || logerr "cp failed"
```
Comment (citrus-it) [inline build/hyperfine/build.sh:46]: The binary name needs adjusting here.
```suggestion
    logcmd cp target/release/$PROG $DESTDIR/$PREFIX/bin/$PROG \
        || logerr "cp failed"
```

When I tried building this, I got:
```
ERROR pkglint.dupaction001.1      path opt/ooce/bin/fd is a duplicate delivered by pkg:/ooce/util/hyperfine@1.10.0,5.11-151034.0 pkg://extra.omnios/ooce/util/fd@7.3.0,5.11-151034.0:20200329T132943Z under all variant combinations
```

File: build/hyperfine/build.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+PROG=hyperfine
+VER=1.10.0
+PKG=ooce/util/hyperfine
+SUMMARY="benchmarking tool"
+DESC="$PROG is a command-line benchmarking tool that provides statistics about command execution time."
+
+BUILD_DEPENDS_IPS=ooce/developer/rust
+
+set_arch 64
+set_checksum none
```
Comment (citrus-it) [inline build/hyperfine/build.sh:29]: Once it's on the mirror, there will be a checksum so please drop this line
```suggestion
```

File: build/hyperfine/build.sh [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+SKIP_LICENCES=Various
```
Comment (citrus-it) [inline build/hyperfine/build.sh:31]: The licence check works fine so you can drop this line too.
```suggestion
```

**Review by citrus-it — CHANGES_REQUESTED** [review]
Thanks for the pull request, this looks like a useful utility to add.
I've put the source on our mirror now, can you please address these four small things.

Thanks!

**Comment by The-King-of-Toasters** [comment]
done

**Review by hadfl — APPROVED** [review]

**Review by citrus-it — APPROVED** [review]

**Pull request merged**